tests/test_pr.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import torch
+
+from piq import PR
+
+
+@pytest.fixture(scope='module')
+def features_y_normal() -> torch.Tensor:
+    return torch.rand(1000, 20)
+
+
+@pytest.fixture(scope='module')
+def features_x_normal() -> torch.Tensor:
+    return torch.rand(1000, 20)
+
+
+@pytest.fixture(scope='module')
+def features_x_beta() -> torch.Tensor:
+    m = torch.distributions.Beta(torch.FloatTensor([2]), torch.FloatTensor([2]))
+    return m.sample([1000, 20]).squeeze()
+
+
+@pytest.fixture(scope='module')
+def features_x_constant() -> torch.Tensor:
+    return torch.ones(1000, 20)
+
+
+def test_initialization() -> None:
+    try:
+        PR()
+    except Exception as e:
+        pytest.fail(f"Unexpected error occurred: {e}")
```

Comment:
I'll vote to drop this test, because metric init with default parameters is checked in all other test

Reply:
True that.